### ISSUE TITLE
Legend  Support Visibility and Values

### ISSUE THREAD

**Issue by aruizhong** [metabase/metabase#45169]
当用户面对多个折线图的趋势时，可以通过点击某个图例取看特定折线图的趋势非常重要。
图例的数值可以帮助用户迅速获得比如 最近一天的值、最大值、平均值等。
这些功能在grafana上都支持，非常期待能在metabase上也看这些功能
https://grafana.com/docs/grafana/latest/panels-visualizations/configure-legend/

**Comment by brunobergher** [comment]
@aruizhong I'm having some difficulty understanding your request.

Are you asking for the ability to see get characteristics of a series (min, max, avg, median) automatically from a click or from the legend?

**Comment by aruizhong** [comment]
> [@aruizhong](https://github.com/aruizhong) I'm having some difficulty understanding your request.
> 
> Are you asking for the ability to see get characteristics of a series (min, max, avg, median) automatically from a click or from the legend?

yes, ![Image](https://github.com/user-attachments/assets/4e1d1779-101f-4381-aff9-e7aac1d85ef1)
here is grafana legend values supportted
